### DISCUSSION TITLE
Can the Hero 11/13 GPS be Souped Up?

### DISCUSSION THREAD

**Discussion by sjmarcy2** [gopro/labs#1187] [Feature Requests]
Decent GPS chip in those cameras, plus IMU.    Can those be turned up...the same GPS chip for instance is used by others at 25hz with more GPS bands in use?   And battery life is not horrible in some of the GPS pucks out there, same basic u-blox chips.  

Not sure if that part of the code is accessible to Labs but if it is, some might have an interest in improved telemetry capability?

**Comment by dnewman-gpsw** [comment]
The GPS is also use multiple bands to get a reliable 10Hz (fastest this chipset can do.) Some older GoPro cameras used a single band at 18Hz, however the reliability and precision wasn't as good.

**Reply by sjmarcy2** [reply]
Per the Hypoxic teardown site, a U-Blox M10S is used in the H13.  Per the spec sheet on the M10S gps, it has a default mode and *also* a High Performance mode.    Around 10 combos of which and how many GPS systems, and which mode of the two modes.    See pages 4 and 5 in here:  

https://content.u-blox.com/sites/default/files/MAX-M10S_DataSheet_UBX-20035208.pdf    

It's a low power module, around 25 mw.  Perhaps for some a few more milliwatts consumption would be okay if there was a return for a given use, JS.  Or maybe it is already maxed out?   

I know that some GPS pucks with the chip choose the best 16 satellites from the many systems and then wind up at 25 hz with great positional accuracy, down to 4 inches, but that is rare.  Around 9-10 inches is common.  See Racebox for example.   

Thanks.

**Reply by dnewman-gpsw** [reply]
Whether it that is the part on not, it is not going to happen. You may have read "Configuration required" for the high performance mode, that is likely more licensing fees or not available in the default configuration. Parts could be flashed to different configurations at the factory. Not something Labs FW can change.

**Reply by sjmarcy2** [reply]
Gotcha, thx.   

Example what can be done in any event, 87 dollar (BF) device, so any licensing is not much :
<img width="366" alt="Screenshot 83" src="https://github.com/user-attachments/assets/91a72af4-1ff0-4c04-a9c9-0cfb5118bfad">

**Reply by dnewman-gpsw** [reply]
25Hz.  5 inch is a nonsense spec., not a real world precision.

**Reply by sjmarcy2** [reply]
I suspect they are referring to relative motion.   As in a vehicle on the move with the GPS *and* IMU used together via some math.   Not pegging a stationary location to 5 inches.   The old accuracy vs precision thing as well.
